docs/docs/icicle/primitives/poseidon.md

```diff
@@ -26,7 +28,8 @@ To generate a secure hash output, the algorithm goes through a series of "full r
 
 ### Full rounds
 
-![Alt text](image-1.png)
+![First Board - 2redone](https://github.com/ingonyama-zk/icicle/assets/122266060/72b54275-d5c5-4b69-b91a-820eed9990a5)
```

Comment:
Is there an image that includes the arrows from each previous stage to the next one?